js/components/search.js

```diff
@@ -5,85 +5,165 @@ $( document ).ready(function() {
-
-            $.each(hit._source.tag, function(i, tag) {
-              tags += '<span class="search-result-list-tag">&nbsp;<a href="../search-results/?q='
-              + tag + '" title="Search for '
-              + tag +'">'
-              + tag + '</a>&nbsp;/</span>';
-            });
-          }
-
-          var content_type = '';
-          if(hit._source.content_type == 'text/html') {
-            content_type = '<span class="glyphicon glyphicon-link"></span> Website';
-          }
-          else if(hit._source.content_type == 'application/pdf') {
-            content_type = '<span class="glyphicon glyphicon-file"></span> PDF';
-          }
-          else {
-            content_type = '<span class="glyphicon glyphicon-file"></span> ' + hit._source.content_type + '';
-          }
-          $('.search-results-container').append('<article class="search-result-list"><h1><a href="'
-            + hit._source.url
-            + '" target="_blank">'
-            + hit._source.title
-            + '</a></h1>'
-            + '<p>'
-            + result_description
-            + '</p>'
-            +'<p>Tagged&nbsp;/'
-            + tags
-            + '</p>'
-            + '</article>');
-        });
-      });
+  /**
+   * Returns URL for search request
+   *
+   * @param {string} query - Raw string holding search terms/query
+   * @param {string} apiKey - Raw string holding API key for api.data.gov
+   * @param {object} [options] - Optional search parameters
+   * @param {number} [options.size=10] - How many results to fetch
+   * @param {number} [options.from=0] - Offset to start from for paging
+   * @returns {string} URL for desired search
+   */
+  function apiSearchUrl( query, apiKey, options ) {
+    var q = encodeURIComponent( query );
+    var size = (options && options.size) || 10;
+    var from = (options && options.from) || 0;
+
+    return (
+      'https://api.data.gov/beckley-federalist/v0/resources/eiti/?' +
+      'q=' + q + '&size=' + size + '&from=' + from + '&api_key=' + apiKey
+    );
+  }
+
+  function localSearchUrl( query, options ) {
+    var q = encodeURIComponent( query );
+    var size = (options && options.size) || 10;
+    var from = (options && options.from) || 0;
+
+    return '/search-results/?q=' + q + '&size=' + size + '&from=' + from;
   }
-  else {
+
+  /**
+   * Returns HTML string of Glyph for given content type
+   *
+   * @param {string} contentType - given document content type
+   * @returns {string} glyph and label for content type or generic default
+   */
+  function getContentTypeGlyph(contentType) {
+    var contentTypes = {
+      'text/html':       '<span class="glyphicon glyphicon-link"></span> Website',
+      'application/pdf': '<span class="glyphicon glyphicon-file"></span> PDF',
+      'default':         '<span class="glyphicon glyphicon-file"></span> ' + contentType
+    };
+
+    return contentTypes[contentType]
+      ? contentTypes[contentType]
+      : contentTypes['default']
+  }
```

Comment:
Not sure why this exists. It wasn't being used before but I went ahead and refactored it into a function just in case there were plans to start adding the glyphs in the future. If the glyph isn't going to be added, this can be zapped entirely.